### DISCUSSION TITLE
add amd k5 processor

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#4472] [Ideas]
freqvencies
75 mhz 
90 mhz 
100 mhz 
105 mhz
116.6 mhz
133 mhz

**Comment by Torinde** [comment]
If you're going after the remaining CPUs using buses from boards already emulated:
- Cyrix 486 (various models, both for 386 boards and 486 boards)
- VIA C3 (P6 bus / Socket 370) - some of its Eden variants start at 300MHz; has models which differ also in the instructions supported (e.g. CMOV)
  - is the currently emulated "Cyrix III" based on WinChip or Cyrix core? In either case, the other one can be added
- various more obscure ones for [486 and earlier](https://github.com/86Box/86Box/discussions/4369) (UMC, C&T, etc.)

**Reply by ghost** [reply]
Explain it more to me. I am sorry for not understanding it

**Reply by Torinde** [reply]
Because you wrote this and #4471 I thought you're asking for emulation of those CPUs that are compatible with the mainboards emulated by 86Box.

Above I mention the remaining other two sets of such CPUs.
